### ISSUE TITLE
WSL Ubuntu - Slint Language Server crashed

### ISSUE THREAD

**Issue by lplath** [slint-ui/slint#5039]
## Platform
WSL:
````
WSL-Version: 2.1.5.0
Kernelversion: 5.15.146.1-2
WSLg-Version: 1.0.60
MSRDC-Version: 1.2.5105
Direct3D-Version: 1.611.1-81528511
DXCore-Version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows-Version: 10.0.22631.3296
````
Ubuntu:
````
Ubuntu 20.04.6 LTS
````
## Steps to reproduce
````
cargo generate --git https://github.com/slint-ui/slint-rust-template --name memory
cd memory
cargo build
cargo run
````

## Error
````
Io error: Broken pipe (os error 32)
Io error: Broken pipe (os error 32)
Io error: Broken pipe (os error 32)
Error: Other("Error running winit event loop: Exit Failure: 1")
````

**Comment by tronical** [comment]
Duplicate of #4089

**Comment by tronical** [comment]
These errors happen when Weston, running inside the [WSLg2 system distroy](https://github.com/microsoft/wslg?tab=readme-ov-file#wslg-architecture-overview) crashes. We've seen this reported before but we also don't know why it crashes - it's a bit out of our hands. I'm hoping that future releases of WSLg fix this. In the meantime, one workaround is to not use Wayland but fall back to X11, by unsetting the `WAYLAND_DISPLAY` environment variable before running the application.
